packages/admin/src/Pages/Page.php

```diff
@@ -174,6 +174,11 @@ protected function getHeaderWidgets(): array
         return [];
     }
 
+    protected function getVisibleHeaderWidgets(): array
+    {
+        return static::getVisibleWidgets($this->getFooterWidgets());
```

Comment:
shouldn't `getFooterWidgets` be `getHeaderWidgets`?

Reply:
Without having written the code, I would guess so. I don't think there is any reason to add complexity to also add extra checks for footer widgets, as the empty space won't be noticed there.

Reply:
I just mean; the `getVisibleHeaderWidgets` method is returning the filtered footer widgets.

Reply:
Whoops, my mind is half-focused on some other code, sorry. Yeah, you're right, while the fix works I don't think this should return `getFooterWidgets()` either.

Reply:
Good catch @maartenpaauw. Copy-paste issue. 😅